articles/urls.py

```diff
@@ -5,13 +5,16 @@
 
 from articles import views
 
-router = SimpleRouter()
-router.register(
+v0_router = SimpleRouter()
+v0_router.register(
     r"articles",
     views.ArticleViewSet,
     basename="articles",
 )
 
+app_name = "articles"
```

Comment:
These URLs have two nested namespaces:
- `include((v0_router.urls, "v1")))` gives namespace `v1`
- when `articles.urls` is included in another URL conf (e.g., the main url conf `open_discussions/urls.py`) the URLs will get additional namspace from `app_name = 'articles'`.

The resulting URLs have names like `articles:v1:articles-list`.

**We could omit the `app_name` portion**. If we do everything works fine. But if `app1/urls.py` and `app2/urls.py` use the same namespaces and both urlconfs are included, then Django gives warnings like
```
WARNINGS:
?: (urls.W005) URL namespace 'v0' isn't unique. You may not be able to reverse all URLs in this namespace
```
The warning is just that—a warning. Everything is OK as long as the url names are unique. But to avoid the warning, I added the `app_name` namespaces.

**Pro:**
- no warning
**Con:**
- url names are more verbose

Happy to drop the `app_name` if people don't like it.